cmd/keycmd/transfer.go

```diff
@@ -931,9 +946,11 @@ func cToPSend(
 	usingLedger bool,
 	amount uint64,
 ) error {
+	ctx, cancel := sdkutils.GetTimedContext(constants.WalletCreationTimeout)
```

Comment:
why not just use getlargeapicontext here

Reply:
because wallet creation can retrieve P/X/C utxos for an account and depending on the account, the operation
may take its time